content/posts/drop-support-for-subversion-mercurial-bazaar.md

```diff
@@ -0,0 +1,72 @@
+title: Drop support for Subversion, Mercurial and Bazaar
+date: 2024-02-19
+category: Changelog
+tags: builders, deprecation
+authors: Manuel Kaufmann
+status: published
+
+
+We are announcing the dropping of support for Bazaar, Mercurial and Subversion version system controls starting on **Monday, June 3rd, 2024**.
+This means we will only support Git to do the initial clone and build your project's documentation.
+
+We've made this hard decision because [99% of our users](https://github.com/readthedocs/readthedocs.org/issues/8840) use Git
+and we can't cover the maintenance cost we were spending on Bazaar, Mercurial and Subversion for a handful amount of projects.
+Besides, those VCSs are not feature-complete in our platform compared to Git and the service provided is degraded for them.
+
+## Deprecation timeline
+
+We understand this change will affect some of our users, so we have a timeline to communicate this deprecation to our users effectively.
+
+* **Monday, April 1, 2024**: Do the first brownout (temporarily enforce this deprecation) for 12 hours: 00:01 PST to 11:59 PST (noon)
+* **Monday, May 6, 2024**: Do a second brownout (temporarily enforce this deprecation) for 24 hours: 00:01 PST to 23:59 PST (midnight)
+* **Monday, May 20, 2024**: Do a third and final brownout (temporarily enforce this deprecation) for 48 hours: 00:01 PST to May 21, 2024 23:59 PST (midnight)
+* **Monday, June 3, 2024**: Fully remove support for building documentation using “build.image” on the configuration file
+
+
+## Possible workarounds
+
+Unfortunately, there is no direct workaround on the Read the Docs side that you can follow to keep building your documentation using these VCSs.
+However, we consider one of the following options can be worth to consider:
```

Comment:
I would say this was pointed out pretty directly with the deprecation description above, otherwise we wouldn't need a deprecation. I'd also say that switching to Git is a solution that _does_ retain all the features, 

So, I'd probably still suggest just not mentioning the first part, or you could reference switching to Git as the suggested fix (though this likely is not an option for anyone still using an unsupported vcs)

Reply:
> So, I'd probably still suggest just not mentioning the first part, 

OK. I will accept your suggestion 👍🏼 

> or you could reference switching to Git as the suggested fix (though this likely is not an option for anyone still using an unsupported vcs)

This is the first possible workaround mentioned: "Convert your repository to Git" 👍🏼

Reply:
Yup, I mean to say that switching to Git is more of a _fix_ than it is a _workaround_ -- "workaround" might imply the project is doing something not quite supported.